examples/4337-gas-metering/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     /* Completeness */
     "skipLibCheck": true /* Skip type checking all .d.ts files. */,
     "outDir": "./dist" /* Redirect output structure to the directory. */,
-    "noImplicitReturns": true /* Report error when not all code paths in function return a value. */,
+    "noImplicitReturns": true /* Report error when not all code paths in function return a value. */
```

Comment:
Personally, I'm a fan of trailing commas - they make future changes 1 line diffs instead of 2 line diffs.

Also `tsc` accepts JSON 5 which allows trailing commas. Is there a reason to undo this change?

Reply:
Oh I see - can we instead get the CI to ignore this file or process it differently?

Reply:
This change is introduced due to running `npm run fmt -w examples/4337-gas-metering`